### PR TITLE
Add css and js easy customizacion

### PR DESCRIPTION
**Pull request by yamila-moreno**
This PR intends to provide an easy way to extend or modify theme CSS. The PR comes directly from the idea taken from @digitalcraftsman in this thread: https://discuss.gohugo.io/t/how-to-override-css-classes-with-hugo/3033/2

This way, we can leave the theme untouched so it's easier to keep it updated and avoid conflicts. AFAIK, this PR doesn't create new problems to those who have changed the theme.

I hope you find it useful; any comments will be welcome! Thanks for your time :)

### CODE REVIEW

**Comment by kakawait** [comment]
Thank again for contribution. It's a great idea and I ever though about adding such mechanism but didn't find time (priority) to add it.

I will check it tomorrow since I don't have any computer around me ATM.

File: docs/user.md [hunk 1]
```diff
@@ -360,6 +360,8 @@ E.g to display a shortcut to open algolia search window :
 | favicon | Your favicon path |
 | imageGallery | Display an image gallery at the end of a post which have `photos` variables. (false: disabled, true: enabled) |
 | hierarchicalCategories | Define categories will create hierarchy between parents: `categories = ["foo", "bar"]` will consider "bar" a sub-category of "foo". If false it will flat categories. |
+| customCSS | Define files with css that override or extend the theme css; they are expected in `static` folder: `customCSS = ["css/mystyles.css"]. |
+| customJS | Define files with js that override or extend the theme js; they are expected in `static` folder: `customJS = ["js/myscripts.js"]. |
```
Comment (kakawait) [inline docs/user.md]: You forget to close ``` ` ``` at end of 

> `customJS = ["js/myscripts.js"].

File: docs/user.md [hunk 2]
```diff
@@ -360,6 +360,8 @@ E.g to display a shortcut to open algolia search window :
 | favicon | Your favicon path |
 | imageGallery | Display an image gallery at the end of a post which have `photos` variables. (false: disabled, true: enabled) |
 | hierarchicalCategories | Define categories will create hierarchy between parents: `categories = ["foo", "bar"]` will consider "bar" a sub-category of "foo". If false it will flat categories. |
+| customCSS | Define files with css that override or extend the theme css; they are expected in `static` folder: `customCSS = ["css/mystyles.css"]. |
```
Comment (kakawait) [inline docs/user.md]: You forget to close ``` ` ``` at end of 

> `customCSS = ["css/mystyles.css"].

**Comment by yamila-moreno** [comment]
Done!! Thanks for reviewing :D

**Comment by kakawait** [comment]
@yamila-moreno on next version `0.4.0-BETA` there will be an evolution about `customJS` and `customCSS` see https://github.com/kakawait/hugo-tranquilpeak-theme/blob/develop/docs/user.md#add-custom-js-or-css-using-configuration

But old syntax will be supported so you don't need to do anything but if you want to add more attribute you can use the new syntax

**ATTENTION** currently not merged on `master` only on `develop` branch

**Comment by yamila-moreno** [comment]
Very interesting evolution; it'll be easy to adapt to the new system; besides, thanks for the back-compatibility. I'll update as soon as it gets to master :D 

Thank you for your effort!

**Pull request merged**